WcaOnRails/app/assets/javascripts/persons.js

```diff
@@ -97,4 +97,12 @@ onPage('persons#show', function() {
     $('.event-selector #radio-' + eventId).click();
     scrollToTabs();
   });
+
+  /* Highlight rows when user clicks on medal count. */
+  $(".highlight-medal").on('click', function(event) {
+    var dataPlace = $(this)[0].getAttribute('data-place');
```

Comment:
I think this should be either `this.getAttribute('data-place')` or `$(this).data('place')`

Reply:
This was working fine but I have changed it to $(this).data('place').